src/Core/Manifest/ModuleResourceLoader.php

```diff
@@ -103,12 +105,47 @@ public function resolveResource($resource)
         $resource = $matches['resource'];
         $moduleObj = ModuleLoader::getModule($module);
         if (!$moduleObj) {
-            throw new InvalidArgumentException("Can't find module '$module'");
+            $append = '';
+            if (!$this->hasModuleComposerConfig($module)) {
+                $append = ", the composer.json file appears to be missing.";
+            }
+            throw new InvalidArgumentException("Can't find module '$module'$append");
```

Comment:
we could just extend the message with "make sure the composer.json file exists" without the extra check/guessing, what do you think?

Reply:
Yeah, agree, lol.

Reply:
+1 but it's not necessarily _because_ the file doesn't exist - let's make sure we use wording that implies it might be missing and it's worth checking

Reply:
Done